core/src/test/scala/akka/persistence/jdbc/serialization/StoreOnlySerializableMessagesTest.scala

```diff
@@ -54,7 +56,7 @@ abstract class StoreOnlySerializableMessagesTest(config: String, schemaType: Sch
   }
 
   override def beforeAll(): Unit = {
-    dropCreate(schemaType)
+    Await.result(SchemaUtils.dropAndCreate(system), 2.seconds)
```

Comment:
why so tight timeout?

Reply:
I'll can increase, but I find it high already.

Reply:
Such a waste to spend time on flaky tests in slow travis or similar env. It will not test better by a short timeout here.